src/main/core/Controller/APINew/User/RegistrationController.php

```diff
@@ -165,6 +165,7 @@ public function initializeAction(Request $request): JsonResponse
                 'locale' => $request->getLocale(),
                 'forceOrganizationCreation' => $this->config->getParameter('registration.force_organization_creation'),
                 'allowWorkspace' => $this->config->getParameter('registration.allow_workspace'),
+                'allowOrganizationSelection' => $this->config->getParameter('registration.allow_organization_selection'),
```

Comment:
Can you expose the new option in the ui parameters please ? https://github.com/claroline/Claroline/blob/13.x/src/main/core/Resources/modules/administration/community/parameters/components/parameters-tab.jsx#L56